### PR TITLE
jest template cli

### PR DESCRIPTION
**Pull request by johnwickers**

### CODE REVIEW

File: cli/templates/app-blank/package.json [hunk 1]
```diff
@@ -4,10 +4,11 @@
   "description": "",
   "scripts": {
     "build": "node scripts/build.js",
+    "serve": "node server/node/server.js",
+    "serve:debug": "node --inspect-brk server/node/server.js",
```
Comment (briandipalma) [inline cli/templates/app-blank/package.json]: Your merge seems to have gone awry, we are moving to using `npm start` to start the application up and away from `npm run serve`.

File: cli/templates/app-blank/package.json [hunk 2]
```diff
@@ -16,11 +17,13 @@
     "@caplin/webpack-config-app": "^4.0.0",
     "babel-preset-env": "^1.2.2",
     "babel-preset-react": "^6.23.0",
-    "chai": "^4.0.0",
-    "karma": "^1.0.0",
-    "karma-chrome-launcher": "^2.0.0",
-    "karma-mocha": "^1.0.0",
-    "karma-webpack": "^2.0.0",
-    "mocha": "^3.0.0"
+    "chai": "^4.0.0"
```
Comment (briandipalma) [inline cli/templates/app-blank/package.json]: Are we still using `chai`? I thought Jest provided its own matchers etc?

File: cli/templates/app-blank/package.json (hunk 1 above)
Comment (johnwickers) [inline cli/templates/app-blank/package.json]: good spot, fixed

File: cli/templates/app-blank/package.json (hunk 2 above)
Comment (johnwickers) [inline cli/templates/app-blank/package.json]: fixed

**Comment by briandipalma** [comment]
Can you publish a new version of the caplin-cli?

**Pull request merged**